drf_tweaks/autodoc.py

```diff
@@ -170,6 +170,32 @@ def _generate_text(cls, documented_cls, method_name):
         return text
 
 
+class OnDemandFieldsAutodoc(AutodocBase):
+    @classmethod
+    def _generate_yaml(cls, documented_cls, method_name):
```

Comment:
Hmm, but yeah, I will get rid of that, but first I will merge the "new swagger" PR.